templates/osbs-buildroot-imagestream.yml.j2

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    openshift.io/image.insecureRepository: "true"
+  name: osbs-buildroot
+  namespace: {{ osbs_namespace }}
+spec:
+  dockerImageRepository: {{ osbs_builldroot_repository }}
```

Comment:
Typo, should be `osbs_buildroot_repository`

Reply:
ok